src/plugins/irc-events/message.js

```diff
@@ -126,6 +126,14 @@ module.exports = function (irc, network) {
 			}
 		}
 
+		// if highlight exceptions match, do not highlight at all
```

Comment:
This comment should be below, where the `if` is.

As for `cleanMessage`, should we make `client.highlightRegex` also test against it?

Reply:
> As for cleanMessage, should we make client.highlightRegex also test against it?
Actually, why not. Things like inserting a control character into nicknames in bot messages will not affect this, and if you specifically add things to custom highlights, you probably prefer if that highlights regardless of formatting.